MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -3222,6 +3233,7 @@ public void writeToXml(PrintWriter pw1, int indent) {
                 + "</phenotypeProbabilities>");
         MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 1, "tougherHealing", tougherHealing);
         MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 1, "useAtB", useAtB);
+        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent + 1, "useStratCon", useStratCon);
```

Comment:
```suggestion
        MekHqXmlUtil.writeSimpleXMLTag(pw1, indent + 1, "useStratCon", useStratCon);
```

Reply:
It's a single boolean, not a collection of booleans, so I don't really see the point. All the surrounding ones use the same syntax.

Reply:
The point is that all XML writers will be moving to this as part of standardizing our writers. I've not yet gotten to the rest of them here, as not all work without https://github.com/MegaMek/megamek/pull/2581